v4m-container/Dockerfile

```diff
@@ -35,48 +35,46 @@ ENV TERM=xterm-256color
 
 # Download necessary executables
 RUN set -eux; \
-	apk add --no-cache \
-		bash \
-		coreutils \
-		ncurses \
-		curl \
-		git \
-		jq \
-		wget \
-		yq \
-		openssh \
-		procps \
-		openssl \
-	;
+  apk add --no-cache \
+  bash \
+  coreutils \
+  ncurses \
+  curl \
+  git \
+  jq \
+  wget \
+  yq \
+  openssh \
+  procps \
+  openssl \
+  ;
 
```

Comment:
My editor automatically made these format changes. I can revert this if that is preferred